pyproject.toml

```diff
@@ -1,7 +1,8 @@
 [build-system]
 requires = [
     'maturin>=1,<2',
-    'typing-extensions >=4.6.0,!=4.7.0'
+    'typing-extensions >=4.6.0,!=4.7.0',
+    'exceptiongroup >=1.1.0'
```

Comment:
I'm curious why we needed the backport as a build dependency?

Reply:
You're right its not needed, I'd added it here before I found `tests/requirements.txt`, removed!